packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -357,6 +358,45 @@ library ChainAssertions {
         require(vm.load(address(portal), bytes32(uint256(61))) == bytes32(0));
     }
 
+    /// @notice Asserts the OptimismPortalInterop is setup correctly
+    function checkOptimismPortalInterop(
+        Types.ContractSet memory _contracts,
+        DeployConfig _cfg,
+        bool _isProxy
+    )
+        internal
+        view
+    {
+        console.log("Running chain assertions on the OptimismPortalInterop");
```

Comment:
This should call `checkOptimismPortal` and extend with anything different. If there is nothing different, we should not make a new function